### ISSUE TITLE
Project list refactoring features

### ISSUE THREAD

**Issue by Billybobbonnet** [WPRN/website#30]
We will make a complete refactor of the project list to include the feedback of the scientific advisory board, referents, and project contacts. Feel free to suggest features in this ticket @saadilahlou @sluck-IEA @Sdbns

What's identified so far:   
- We should add a mobile only layout and break the table into blocks (https://github.com/WPRN/website/issues/27)
- The status badge (the white/green/star at the beginning of each line) should be moved at the end of the line. It would prepare for the upcoming rating system where we can give several stars. I'm thinking of a classic read-only scale such as [this one](https://vuetifyjs.com/en/components/ratings/)
- Bug fixes (https://github.com/WPRN/website/issues/29, https://github.com/WPRN/website/issues/22)
- Add a new project location logic and display (https://github.com/WPRN/website/issues/4, https://github.com/WPRN/website/issues/16)
- Add a pinned mode to the filter panel (https://github.com/WPRN/website/issues/18)
- A new `institution` filter and column should be added, partially replacing the former one
- A `Blazon` column should be added 
- The truncated description of the project in the expanded row should display the first occurrence of a matched text even if it is past the x first characters. It would require to truncate earlier, add an ellipsis, display the matched text + a few words before/after for context, and the read more
- _(edit 05-29-2020)_ Add a block word list to enhance the search (https://github.com/WPRN/website/issues/37)

What should be discussed:
- Better colors for the search highlight
- Font weight and size should be more consistent between the different parts of the list. We can probably remove some weight, adapt dynamically the size and overall get a cleaner result while keeping a good legibility.
- Maybe add an instant search on the full text search input
- Improve the skeleton loader to make it more similar to actual content
- Add an expand/collapse transition to expanded row

**Comment by sluck-IEA** [comment]
This all seems like very sensible updates. I am just very skeptical about a star scale which would risk change the whole spirit of WPRN (and what about all the projects with no stars? it would make them appear really bad). I think the starring system should be well thought through, in order not to make WPRN look like a generic commercial website. Is it necessary to have more than one star? If so how many? I am not sure than more thatn two are necessary. Should it be several stars or different icons?

**Comment by Billybobbonnet** [comment]
I agree about the fact that the starring system should be well thought through, and we indeed could keep the single star instead of switching to multiple stars. Referents will have a real scale (not a binary toggle) to assess project quality however.

The current result order works as follows:
- First sort: featured > verified > inactive > non-verified (we'll see about which comes first between inactive & non-verified)
- Among a status (e.g. verified), we will use the project score as tie breaker (second sort).
